i18n/en.i18n.json

```diff
@@ -200,6 +200,7 @@
   "Room" : "Room",
   "Room_deleted" : "Room deleted",
   "Room_deleted_by" : "Room deleted by: <em>__user_by__</em>",
+  "Room_label_changed" : "Security label changed to: <em>__label__</em> by: <em>__user_by__</em>",
```

Comment:
I won't be anal, but I would recommend having a class="label" and css in the css file :(
HTML
<em class="label">__label__</em>

In CSS
.label {
  text-transform: uppercase;
}